src/librustc_middle/ty/layout.rs

```diff
@@ -2283,10 +2283,19 @@ impl<'tcx> ty::Instance<'tcx> {
     fn fn_sig_for_fn_abi(&self, tcx: TyCtxt<'tcx>) -> ty::PolyFnSig<'tcx> {
         let ty = self.monomorphic_ty(tcx);
         match ty.kind {
-            ty::FnDef(..) |
-            // Shims currently have type FnPtr. Not sure this should remain.
-            ty::FnPtr(_) => {
```

Comment:
Ah, heh, I see, this kind of comment between patterns stops `rustfmt` from working, and you removed it.

Reply:
Resolving as I don't think there's anything actionable in this comment.